source/howto/AddUsersOpenStackVMInstance.rst

```diff
@@ -0,0 +1,148 @@
+##################################################
+Create additional user(s) on OpenStack VM/Instance
+##################################################
+
+Introduction
+############
+
+This guide explains how to create users in Linux using the command line and the *useradd* command and  grant administrator privileges using *sudo*.
```

Comment:
Using `useradd` is not best practice, the `useradd` sbin will do the bare minimum (which is fine for service accounts but not in this case), whereas `adduser` will also setup quota, home directories associated permissions and groups...etc.

https://unix.stackexchange.com/a/121072

Can we swap this out with adduser